torch/testing/_internal/subclasses.py

```diff
@@ -40,9 +39,6 @@ def __tensor_flatten__(self):
     def __tensor_unflatten__(inner_tensors, meta, outer_size, outer_stride):
         assert meta is None
         a = inner_tensors["a"]
-        if is_fake(a):
-            assert outer_size is not None
-            assert outer_stride is not None
```

Comment:
what's the reason for this change?

(IIRC, it made our testing a bit more robust, because we wanted to make sure that at compile time we were properly passing in distinct outer_size/outer_stride arguments during tracing, to ensure we were handling dynamic shapes correctly)

Reply:
is_fake(a) fails to run with TEST_WITH_DYNAMO.
I wanted to have not only TwoTensor subclass in test, but also WrapperSubclass. And removed this to use WrapperSubclass.

Ok, I will return this and remove WrapperSubclass from the test. We can solve is_fake dynamo thing later.